src/main/java/dev/emi/trinkets/mixin/LivingEntityMixin.java

```diff
@@ -52,6 +53,7 @@
  *
  * @author Emi
  */
+@SuppressWarnings("UnreachableCode")
```

Comment:
SuppressWarnings also unneeded here.

Reply:
Added it purely for dev, as my ide marked all code post casting as unreachable, but can remove that